decred/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "decred"
+version = "0.1.0"
+description = "A Python 3 Decred toolkit."
```

Comment:
The `Decred` package is available only to Python3?

Reply:
> The Decred package is available only to Python3?

Yes, I don't see a strong enough reason to support Python 2 at this point.

> On the requisites there's python^3.7. Shouldn't this be on the description (instead of only 3)?

We could be more precise, yes.

> The most common Python3 which comes with the great variety of Linux distros is python3.5

We can extend the automated testing to Python 3.5 and 3.6, and we should probably add 3.8 too. The action definition supports this already, but it will make the PR checks take more time.

Reply:
Python 2 support [ended on Jan 1st](https://www.python.org/doc/sunset-python-2/). We can just say **Python** now, as far as I'm concerned. We should probably support back to 3.5 at least.

Reply:
Adding 3.5, 3.6 and 3.8, let's see what happens.

Reply:
Cannot use Black and formatted string literals in 3.5, and who knows what else. Setting the Python minimum supported version at 3.6.